getting_started/administrators.adoc

```diff
@@ -46,6 +46,13 @@ Choose one of the following installation methods that works best for you.
 You can quickly get OpenShift running in a Docker container using images from
 https://hub.docker.com[Docker Hub] on a Linux system.
 
+[CAUTION]
+====
+OpenShift Origin requires access to port 7001. If a service (such as the 
+Kubernetes service) is running and occupying this port, stop that service 
```

Comment:
Suggest for clarity:

If a service on your host (such as another Kubernetes or etcd service) is already running and occupying this port locally, [...]